app/views/govuk_publishing_components/components/docs/heading.yml

```diff
@@ -68,3 +68,7 @@ examples:
       brand: 'department-for-environment-food-rural-affairs'
       padding: true
       border_top: 5
+  with_lang_attribute:
+    data:
```

Comment:
Can we note in the description for this that the `lang` attribute needs to be a valid BCP47 string? I've linked to https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang#Language_tag_syntax before.